modules/rosa-operator-config.adoc

```diff
@@ -16,25 +16,34 @@ When using a {hcp-title} cluster, you must create the Operator IAM roles that ar
 
 .Procedure
 
+* Set your prefix name to an environment variable using the following command:
++
+[source,terminal]
+----
+$ OPERATOR_ROLES_PREFIX=<prefix-name>
+----
 * To create your Operator roles, run the following command:
+
 +
 [source,terminal]
 ----
-$ rosa create operator-roles --hosted-cp --prefix <prefix-name> --oidc-config-id <oidc-config-id>
+$ rosa create operator-roles --hosted-cp --prefix $OPERATOR_ROLES_PREFIX --oidc-config-id $OIDC_ID --installer-role-arn ${ACCOUNT_ROLES_PREFIX}-HCP-ROSA-Installer-Role
 ----
 +
 The following breakdown provides options for the Operator role creation.
 +
 [source,terminal]
 ----
 $ rosa create operator-roles --hosted-cp
-	--prefix <prefix-name> <1>
-	--oidc-config-id <oidc-config-id> <2>
+	--prefix $OPERATOR_ROLES_PREFIX <1>
+	--oidc-config-id $OIDC_ID <2>
+	--installer-role-arn ${ACCOUNT_ROLES_PREFIX}-HCP-ROSA-Installer-Role <3>
```

Comment:
The  --installer-role-arn format should like this
arn:aws:iam::<account_id>:role/${ACCOUNT_ROLES_PREFIX}-HCP-ROSA-Installer-Role

Reply:
replaced with:
"--installer-role-arn $arn:aws:iam::${AWS_ACCOUNT_ID}:role/${ACCOUNT_ROLES_PREFIX}-HCP-ROSA-Installer-Role"